presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -284,6 +284,9 @@ public void testMaps()
                 .replaceAll("test_maps", "lineitem_ex")
                 .replaceAll("cardinality", "array_length");
         try {
+            //filter on nested columns
+            assertQueryUsingH2Cte("SELECT * FROM test_maps WHERE map_keys[1] > 10 and map_keys[1] < 20", rewriter);
```

Comment:
just realized this change. What is `H2Cte`?

Reply:
the assertQueryUsingH2Cte is added as h2 query required some rewriting as it does not match the same query as in presto